bootcamp/RAG/zilliz_bento_RAG.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    COLLECTION_NAME = "bmo_test"
</pre><p>what is bmo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I fix it to "Bento_Zilliz_RAG"